src/main/kotlin/no/nav/familie/ba/sak/kjerne/steg/VurderTilbakekrevingSteg.kt

```diff
@@ -47,8 +53,12 @@ class VurderTilbakekrevingSteg(
                 if (finnesPerioderMedEtterbetalingStørreEnnMaksBeløp(behandlinId = behandling.id)) kastException(behandling)
             }
             else -> {
-                val finnesEtterbetaling = simuleringService.hentEtterbetaling(behandlingId = behandling.id) != BigDecimal.ZERO
-                if (finnesEtterbetaling) kastException(behandling)
+                val finnesEtterBetaling = hentTotalEtterbetalingFørMars2023(behandling.id) != BigDecimal.ZERO
+                if (finnesEtterBetaling &&
+                    !erPositivePerioderesultaterPåMaks1KronePerBarnOgTotaltAvvikUnderBeløpsgrense(behandling.id)
```

Comment:
Denne er en liten munnfull. Jeg måtte lese denne flere ganger før jeg skjønte hva som sto her. Hva tenker du med noe lignende dette?
```suggestion
                val simuleringsperioderFørMars2023 = hentSimuleringsperioderFørMars2023(behandling.id)
                if (finnesEtterBetaling &&
                    simuleringsperioderFørMars2023.harKunPositiveResultater() &&
                    simuleringsperioderFørMars2023.harMaks1KroneIResultatPerBarn() &&
                    simuleringsperioderFørMars2023.harTotaltAvvikUnderBeløpsgrense()
```

Reply:
Enklere å lese det, ja. Fikser:)